src/styles/utils/variables.scss

```diff
@@ -0,0 +1,12 @@
+$start-day:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+$paddings: 10px;
```

Comment:
It is better to give a clearer name so that it is immediately clear what exactly these `paddings` are for.